pkg/lib/bundle/relations.json

```diff
@@ -1511,5 +1511,15 @@
     "name": "Target space id",
     "readonly": true,
     "source": "derived"
+  },
+  {
+    "description": "Last time object type was used",
+    "format": "date",
+    "hidden": true,
+    "key": "lastUsedDate",
+    "maxCount": 1,
+    "name": "Last Used date",
+    "readonly": true,
+    "source": "local"
```

Comment:
it should be local or synced?

Reply:
We decided with @requilence that is is better to be local, as we fisrt don't need to sync this detail between devices, and second particular devices will have their own type sorting statistics